packages/lib/CalendarService.ts

```diff
@@ -275,7 +298,7 @@ export default abstract class BaseCalendarService implements Calendar {
     const events: { start: string; end: string }[] = [];
 
     objects.forEach((object) => {
-      if (object.data == null) return;
+      if (object.data == null || JSON.stringify(object.data) == "{}") return;
```

Comment:
when the object doesn't contain any data, we return. There will be a follow up on this condition to consider situations when the service providers **do not** send data directly, but that's a solution to a different problem and outside of the scope of this PR.